Tests/LibMarkdown/TestCommonmark.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, Peter Elliott <pelliott@ualberta.ca>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <AK/JsonParser.h>
+#include <AK/String.h>
+#include <LibCore/File.h>
+#include <LibMarkdown/Document.h>
+#include <LibTest/TestCase.h>
+#include <LibTest/TestSuite.h>
+
+TEST_SETUP
+{
+    auto file = Core::File::construct("./Markdown/commonmark.spec.json");
```

Comment:
The code and configuration looks pretty good, but one issue is that this file is not installed on the target.

And this test does actually run on the target and those failures cause CI failures :)

https://github.com/SerenityOS/serenity/runs/3456920814?check_suite_focus=true#step:23:322

So I think a new cmake file in Meta/CMake that downloads the commonmark spec if it's not downloaded already and does an install(FILES) for it might be in order? so that `run-tests` can succeed.

Reply:
`TestCommmark` now works in serenity, and is disabled by default